actionview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Support `:assigns` option with `ActionView::Rendering#_render_template`.
```

Comment:
Hm, should we then mention that `ActionController::Base#render/render_to_string` support `:assigns`?